lib/importmap/commands.rb

```diff
@@ -28,7 +28,7 @@ def pin(*packages)
         if packager.packaged?(package)
           gsub_file("config/importmap.rb", /^pin "#{package}".*$/, pin, verbose: false)
         else
-          append_to_file("config/importmap.rb", "#{pin}\n", verbose: false)
+          append_to_file("config/importmap.rb", "\n#{pin}\n", verbose: false)
```

Comment:
This will leave blank lines between pins, if you run the pin command twice or more.

Could you add the newline only if there isn't one at the end of the file?